Sources_v3/StreamChatUI/UIConfig.swift

```diff
@@ -106,6 +107,20 @@ public extension UIConfig {
     }
 }
 
+public extension UIConfig {
+    struct Font {
+        public let captionBold: UIFont = .streamCaptionBold
+        public let footnote: UIFont = .streamFootnote
+        public let footnoteBold: UIFont = .streamFootnoteBold
+        public let body: UIFont = .streamBody
+        public let bodyBold: UIFont = .streamBodyBold
+        public let bodyItalic: UIFont = .streamBodyItalic
+        public let headline: UIFont = .streamHeadline
+        public let headlineBold: UIFont = .streamHeadlineBold
+        public let title: UIFont = .streamTitle
```

Comment:
`var` instead of `let`, otherwise it destroys whole point of `UIConfig`

Reply:
Thanks :) This is done.